src/components/PostCard.tsx

```diff
@@ -0,0 +1,35 @@
+const PostCard = ({ post }) => {
+  return (
+    <article className="bg-arg-white p-6 sm:p-8 rounded border-4 border-arg-muted shadow-md shadow-arg-muted/40 grid gap-4">
+      <div>
+        <time
+          dateTime={new Date(post.data.date).toISOString()}
+          className="uppercase text-arg-gold"
+        >
+          {new Date(post.data.date).toLocaleDateString("en-us", {
```

Comment:
Not sure if case-sensitive: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString#using_locales

```suggestion
          {new Date(post.data.date).toLocaleDateString("en-US", {
```